ansible_ai_connect/ai/api/model_pipelines/wca/pipelines_saas.py

```diff
@@ -427,9 +427,6 @@ class WCASaaSRoleExplanationPipeline(
     def __init__(self, config: WCASaaSConfiguration):
         super().__init__(config=config)
 
-    def invoke(self, params: RoleExplanationParameters) -> RoleExplanationResponse:
```

Comment:
This overrode the super class's implementation and was not needed.

Reply:
👍